.github/workflows/consuming.yml

```diff
@@ -36,11 +36,11 @@ jobs:
           # Test multiple K8s versions only in submariner-operator, balancing coverage and jobs
           # Recentness of K8s versions are limited by kindest/node image releases
           - project: submariner-operator
-            k8s_version: 1.18.15
+            k8s_version: 1.18.19
           - project: submariner-operator
-            k8s_version: 1.19.7
+            k8s_version: 1.19.11
           - project: submariner-operator
-            k8s_version: 1.20.2
+            k8s_version: 1.20.7
```

Comment:
Shouldn't these be short versions? :)

Reply:
They should now ;-). I wrote this patch before filing #567 as a result of the resulting downstream failures!